integration-tests/src/test/java/oracle/kubernetes/operator/BaseTest.java

```diff
@@ -520,4 +583,29 @@ private void callWebAppAndVerifyScaling(Domain domain, int replicas) throws Exce
       TestUtils.checkServiceCreated(podName, domainNS);
     }
   }
+
+  public static void tearDown() throws Exception {
+    StringBuffer cmd =
+        new StringBuffer("export RESULT_ROOT=$RESULT_ROOT && export PV_ROOT=$PV_ROOT && ");
+    cmd.append(BaseTest.getProjectRoot())
+        .append("/integration-tests/src/test/resources/statedump.sh");
+    logger.info("Running " + cmd);
+
+    ExecResult result = ExecCommand.exec(cmd.toString());
+    if (result.exitValue() == 0) {
+      logger.info("Executed statedump.sh " + result.stdout());
+    } else {
+      logger.info("Execution of statedump.sh failed, " + result.stderr() + "\n" + result.stdout());
+    }
+
+    // if (JENKINS) {
+    result = cleanup();
+    logger.info("cleanup result =" + result.stdout() + "\n " + result.stderr());
```

Comment:
Make sure statedump and cleanup both try to renew the lease before they start (and exit with a failure if the lease doesn't renew).

Reply:
Or renew the lease right before calling each...

Reply:
Done